src/main/java/techreborn/blocks/machine/tier1/ResinBasinBlock.java

```diff
@@ -82,6 +88,19 @@ public Direction getFacing(BlockState state) {
 		return state.get(FACING);
 	}
 
+	/*@SuppressWarnings("deprecation")
+	@Override
+	public ActionResult onUse(BlockState state, World world, BlockPos pos, PlayerEntity player, Hand hand, BlockHitResult hit) {
```

Comment:
Is this needed? Generally dont want to commit commented out code?

Reply:
The idea is that it *should* work, but it doesn't. I don't want to commit a bug, but I want people to be able to solve the bug, because I am unlikely to follow it further.